privacyidea/static/components/user/controllers/userControllers.js

```diff
@@ -25,19 +30,102 @@ angular.module("privacyideaApp")
                                                gettextCatalog){
 
         $scope.formInit = {};
+
         ConfigFactory.getEditableResolvers(function (data){
             var resolvers = data.result.value;
             var resolvernames = [];
+            var userAttributes = [];
             for (var rname in resolvers) {
                 resolvernames.push(rname);
             }
             $scope.formInit.resolvernames = resolvernames;
             if (resolvernames.length > 0) {
                 $scope.resolvername = resolvernames[0];
             }
+
+            userAttributes = getUserAttributes(data);
+            $scope.userAttributes = userAttributes;
+            $scope.getAddUserAttributes($scope.resolvername);
+
         });
 
+        function getUserAttributes(data) {
+            var resolvers = data.result.value;
+            var allResolverAttributes = [];
+
+            for (var rname in resolvers) {
+                var resolver = resolvers[rname];
+                switch (resolver.type){
+                    case "ldapresolver":
+                        var userinfo = JSON.parse(gettextCatalog.getString(resolver.data.USERINFO));
```

Comment:
I have not checked this. I the gettextCatalog.getString realy necessary?
The data.USERINFO and data.MAP is already a string.